app/src/main/res/layout/activity_main.xml

```diff
@@ -13,9 +13,42 @@
         android:layout_marginStart="117dp"
         android:layout_marginTop="409dp"
         android:layout_marginEnd="118dp"
-        android:text="Trigger Permissions"
+        android:text="@string/trigger_permissions"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <EditText
```

Comment:
since you only want to display text, you should use a TextView instead of EditText

Reply:
fixed on latest commit